### ISSUE TITLE
inconsistent resizing in list view

### ISSUE THREAD

**Issue by juliagoolia** [sfbrigade/support-sfusd#224]
if you scroll down through the list view (both desktop and mobile) and open the school cards, sometimes they open consistently, but sometimes (especially if you scroll to the bottom and then expand them as you scroll up) they don't. here's a link to a recording: https://sfbrigade.slack.com/files/U01JYKWN43D/F07L5J83AQ2/rpreplay_final1725567192.mp4

**Comment by thomhickey** [comment]
Hi @juliagoolia ... seems to work as expected to me and I can't tell from the video which schools you are clicking/tapping on so it's hard to see what the problem is there as well. Happy to fix this once I can reproduce it!

**Comment by thomhickey** [comment]
@juliagoolia yeah I can get it to behave wonky, but it takes some doing. most of what i'm seeing is what you noticed in your other bug that the learn more button doesn't scroll into view always (#226) ... which I have fixed in a branch, but i'm not happy with the heavy lifting that was necessary to fix it so i'm fine tuning the code some more.

**Comment by thomhickey** [comment]
hi @juliagoolia  ... is this fixed in PR #228 ?
